app/presenters/flat_reasons_for_rejection_presenter.rb

```diff
@@ -61,12 +61,16 @@ def self.subreason_exists?(reason, subreason)
     reason.include?(subreason)
   end
 
-  def self.title_for_top_level_reason(reason)
+  def self.humanized_title_for_top_level_reason(reason)
```

Comment:
This is a bit of a hack. I didn't want to change the methods on lines 68 and 72 as it would have made the lines at the top of the file even longer than they are now.